website/_enemies/forest/wolf.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: Fox / Wolf
```

Comment:
People still refer to them as Foxes so this needs to be covered in the content below the front matter (`---`'s)